src/main/java/org/kuali/kfs/module/ar/service/impl/ContractsGrantsInvoiceCreateDocumentServiceImpl.java

```diff
@@ -1643,24 +1648,37 @@ protected void populateContractsGrantsInvoiceDocument(
     protected KualiDecimal calculateTotalExpenditureAmount(
             final ContractsGrantsInvoiceDocument document,
             final List<ContractsGrantsLetterOfCreditReviewDetail> locReviewDetails) {
-        final Map<String, KualiDecimal> totalBilledByAccountNumberMap = new HashMap<>();
```

Comment:
Another spot where it would be helpful to indicate that several lines were removed by a backport.